### DISCUSSION TITLE
[Feature Request] Add default classNames on provider

### DISCUSSION THREAD

**Discussion by dexcell** [mantinedev/mantine#1045] [Feature requests]
### Usage example, including component/hook API

So people can add tailwind or css utility classes to override/add button className by default

### Possible implementation – describe how the feature can be implemented

````
<MantineProvider
      classNames={{ Button: { root: 'font-medium bg-red-500' } }}
    >
````

### Do you want to contribute this feature and create a pull request

No

**Comment by frec-hansen** [comment]
#912
